# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/stf/dvdauthor/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -dur dvdauthor-0.6.10/src/dvdvob.c dvdauthor-0.6.10-patch/src/dvdvob.c
--- dvdauthor-0.6.10/src/dvdvob.c	2004-03-14 21:13:26.000000000 +0000
+++ dvdauthor-0.6.10-patch/src/dvdvob.c	2004-07-12 23:24:43.000000000 +0000
@@ -1,4 +1,4 @@
-/*
+/*
  * Copyright (C) 2002 Scott Smith (trckjunky@users.sourceforge.net)
  *
  * This program is free software; you can redistribute it and/or modify
@@ -1148,8 +1148,8 @@
                         printpts(pts1);
                         fprintf(stderr,"\n");
                         ach->audpts[ach->numaudpts-1].pts[1]=pts0;
-                    noshow:
                     }
+                    noshow:
                     ach->audpts[ach->numaudpts].pts[0]=pts0;
                     ach->audpts[ach->numaudpts].pts[1]=pts1;
                     ach->audpts[ach->numaudpts].sect=cursect;
Nur in dvdauthor-0.6.10-patch/src: dvdvob.c~.
